zerver/openapi/zulip.yaml

```diff
@@ -3109,6 +3109,15 @@ paths:
                                   description: |
                                     Array of dictionaries where each dictionary contains details about
                                     a single realm linkifier.
+
+                                    The client will always receive an empty list unless the event queue
+                                    is registered with `linkifier_url_template` client capability set to `true`.
+                                    See [`POST /register`](/api/register-queue#parameter-client_capabilities).
+                                    for how client capabilities can be specified.
+
+                                    **Changes**: New in Zulip 7.0 (feature level 175). Linkifiers that
+                                    use [RFC 6570](https://www.rfc-editor.org/rfc/rfc6570.html) compliant URL template
+                                    are supported.
```

Comment:
This is saying that this property is new in Zulip 7.0.

Instead, it should say what changed. Because the new state of things is already described in the main text of the description (and of the descriptions of properties deeper in the tree), the emphasis should be on clearly describing the old state, doing so in terms of a diff from the new state.

Here, because the change from `url_format` to `url_template` is already covered below, I think the only difference that this particular "Changes" item needs to cover is that the bit about "will always receive an empty list unless" was new in this version.

Reply:
I added "will always receive an empty list unless" to the "changes" section of the event instead of the property.